resources/functionMap.php

```diff
@@ -12668,11 +12678,19 @@
 'uopz_rename\'1' => ['void', 'function'=>'string', 'rename'=>'string'],
 'uopz_restore' => ['void', 'class'=>'string', 'function'=>'string'],
 'uopz_restore\'1' => ['void', 'function'=>'string'],
+'uopz_set_hook' => ['bool', 'class'=>'string', 'class'=>'function', 'hook'=>'Closure'],
+'uopz_set_hook\1' => ['bool', 'class'=>'string', 'hook'=>'Closure'],
 'uopz_set_mock' => ['void', 'class'=>'string', 'mock'=>'object|string'],
+'uopz_set_property' => ['void', 'class'=>'string', 'property'=>'string', 'value'=>'mixed'],
+'uopz_set_property\1' => ['void', 'instance'=>'object', 'property'=>'string', 'value'=>'mixed'],
 'uopz_set_return' => ['bool', 'class'=>'string', 'function'=>'string', 'value'=>'mixed', 'execute='=>'bool'],
 'uopz_set_return\'1' => ['bool', 'function'=>'string', 'value'=>'mixed', 'execute='=>'bool'],
+'uopz_set_return' => ['bool', 'class'=>'string', 'function'=>'string', 'static'=>'array'],
+'uopz_set_return\'1' => ['bool', 'function'=>'string', 'static='=>'array'],
```

Comment:
I might be misunderstanding this, but currently PHPStan is complaining that `Parameter #3 $static of function uopz_set_return expects array` while parameter #3 is actually the value in the following form:
https://github.com/krakjoe/uopz/blob/v6.1.2/uopz.c#L185-L186
```php
 uopz_set_return ( string $class , string $function , mixed $value , bool $execute = false ) : bool
```

Reply:
@SMillerDev Hi, this is a pretty old PR. If you see a buggy behaviour, please open an issue with reproduction, thank you.

Reply:
Will do, thanks.